src/python_x509_pkcs11/ca.py

```diff
@@ -186,11 +184,11 @@ async def create(  # pylint: disable-msg=too-many-arguments,too-many-locals
     subject_name: Dict[str, str],
     signer_subject_name: Optional[Dict[str, str]] = None,
     signer_key_label: Optional[str] = None,
-    signer_key_type: Optional[str] = None,
+    signer_key_type: Optional[Union[str, KEYTYPES]] = None,
```

Comment:
Would it be possible to enforce the use of KEYTYPES in order avoid the need of checking and possibility to choose an unsupported?

Reply:
This is the only case where I can see `singer_key_type` can be a `None` indicating that it should be the same with `key_type` argument. For the unsupported option, check the below comment.